x-team/xp-ui
# Upgrade HeaderBar

**Release Type:** *Non-breaking changes*

Fixes https://x-team-internal.atlassian.net/browse/XP-3369

## Description

Upgrades the HeaderBar to accept links for the new navigation menu. By default it displays as it was before, without link and only the centralised logo.

## Checklist


## Steps to Test or Reproduce

1. Run `npm start` 
1. Check basic usage story (it should look the same as it was previously): https://xpui-develop.netlify.com/?path=/story/ui-components-headerbar--basic-usage
1. Check the with links story: http://localhost:9001/?path=/story/ui-components-headerbar-use-cases--with-links

## Impacted Areas in Application

- HeaderBar component

## Screenshots

![2019-10-22 12 46 03](https://user-images.githubusercontent.com/131859/67305001-12a74000-f4cb-11e9-9dcd-115d9b1d6210.gif)
